Italian/aiuto/chi/chi.md

```diff
@@ -0,0 +1,5 @@
+Sono il supporto tecnico della tua mente. Faccio parte di un programma speciale creato per carpire i
+
+segreti della mente umana.
+
+Mi sà tanto che devo smettere di bere, [Buonanotte!](..\caramelle.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.